### PR TITLE
Colorize some output

### PR DESCRIPTION
**Pull request by mathias-baumann-sociomantic**

### CODE REVIEW

**Comment by mathias-baumann-sociomantic** [comment]
Fixed travis

**Comment by leandro-lucarella-sociomantic** [comment]
I've been using this and it's of great help to visually grep for important info.

Some feeback:

On the general plan it looks weird that only some points are coloured. I understand some items are much more important than others (creating tags, merging), but still makes the list look quite weird.

Maybe it would be better to colour either all the list items (without the ` (command)`) the same and highlight (bold) versions or have a colour code for each action (tagging yellow, checking out cyan, merge violet -like github, wink wink-).

I would also colour what is going to be done right before a question, like:
```
Done! Some references should be pushed: ["v0.11.2", "v0.x.x"]
Would you like to push the modified references now? y/n: y
Some tags on github should be released: ["v0.11.2"]
Would you like to release those tags now? y/n: y
...
Some milestones on github should be closed: ["v0.11.2"]
Would you like to close them now? y/n: y
```
Colour both `Done! Some references should be pushed: ["v0.11.2", "v0.x.x"]` and `Some tags on github should be released: ["v0.11.2"]` and `Some milestones on github should be closed: ["v0.11.2"]`. Maybe again either having both with the same colour (green?) or having one colour for pushes (yellow?) and one for releasing (cyan?) and for closing a milestone (red? like closed issues?).

When closing the milestone, the open issues count is shown in red. I think if the count is 0 it should either not be coloured (the "Closing" word should be probably coloured though, red probably), or should be green, indicating is all good. I would use red only if there are open issues left.

Finally, I would colour the whole e-mail notes with some grey or other very opaque colour, so you know exactly what  is included as part of the e-mail very clearly.

File: Build.mak [hunk 1]
```diff
@@ -17,19 +20,24 @@ $O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
...
+
 ################################################################################
 
-$O/%unittests: override LDFLAGS += -L$O -lvibed -ldyaml -levent -lssl -lcrypto
+$O/%unittests: override LDFLAGS += -L$O -lvibed -ldyaml -levent \
+	-lssl -lcrypto -ld-colorize
 $O/pkg-neptune.stamp: release overview
 
 %unittests: $O/libvibed.a $O/libdyaml.a
+%unittests: $O/libvibed.a $O/libdyaml.a $O/libd-colorize
```
Comment (leandro-lucarella-sociomantic) [inline Build.mak]: This should end with `.a`.

**Comment by leandro-lucarella-sociomantic** [comment]
I guess this is low prio but the idea is to keep working on it? Or do you prefer to merge as is (after rebase) and leave further improvements to the future?

**Comment by mathias-baumann-sociomantic** [comment]
I would keep working on it with low prior for now

**Comment by leandro-lucarella-sociomantic** [comment]
I think we should probably merge this as is, is better than nothing, we can create another issue to improve it further.

**Comment by leandro-lucarella-sociomantic** [comment]
Oh, I tried to adapt this to the current master but tests fail, probably because they are trying to parse some colored output. I guess we need to include some sort of `--no-color` option for that...

**Comment by leandro-lucarella-sociomantic** [comment]
https://github.com/yamadapc/d-colorize/issues/18

File: Build.mak [hunk 2]
```diff
@@ -17,11 +20,15 @@ $O/libdyaml.a: $(shell find $C/submodules/d-yaml/source -type f -name '*.d') \
 		$(shell find $C/submodules/tinyendian/source -type f -name '*.d')
 	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
 
+$O/libd-colorize.a: $(shell find $C/submodules/d-colorize/source -type f -name '*.d')
+	$(call exec, dmd $(DFLAGS) -lib -of$@ $?)
+
 ################################################################################
 
 $O/allunittests $O/fastunittest: $O/libvibed.a $O/libdyaml.a
```
Comment (leandro-lucarella-sociomantic) [inline Build.mak]: This is probably missing `$O/libcolorize.a` too.

**Comment by leandro-lucarella-sociomantic** [comment]
:thinking: I had the feeling I was playing with this and all test were broken because the tests parse output without expecting colour codes, that's why I opened the issue in the d-colorize project.

**Comment by mathias-baumann-sociomantic** [comment]
I haz done a fix

File: Build.mak (hunk 2 above)
Comment (mathias-baumann-sociomantic) [reply Build.mak]: https://github.com/sociomantic-tsunami/neptune/pull/139

**Pull request merged**